testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -423,4 +439,38 @@ public void testSelectInformationSchemaColumns()
                         "('tables'), " +
                         "('views')");
     }
+
+    @Test
+    public void testRenameTableAcrossSchema()
```

Comment:
Can we base `BaseConnectorTest` on `BaseConnectorSmokeTests` somehow? I don't like this copy pasting. If it is not possible today, I would remove them from `BaseConnectorSmokeTests` for now.

Reply:
> Can we base BaseConnectorTest on BaseConnectorSmokeTests somehow? 

sure, if we get rid of ATDQ and ATQ

Reply:
I assume nothing to be done here for now.

Reply:
can we extract method to some helper class for now?